src/utils/createComponent.ts

```diff
@@ -105,4 +108,20 @@ export function createComponent<I extends HTMLElement, E extends EventNames = {}
         }
       : options,
   );
+
+  const originalRenderFunc = (Component as any).render;
+
+  (Component as any).render = (props: any, ...rest: any) => {
```

Comment:
`props` isn't used for anything so you could destructure all the arguments to a single variable.

Reply:
Done.